### ISSUE TITLE
What's is plan for the formal release?

### ISSUE THREAD

**Issue by feikiss** [go-swagger/go-swagger#165]
Hi casualjim, @casualjim 

 Do we have any plan for publishing a formal stable release of Go-Swagger project? 

In the readme, I saw:
"This API is not stable yet, when it is stable it will be distributed over gopkg.in", but I cannot find what's the exact date for the stable release in the readme.

Any comment is appreciated! Thank you.

Thanks,
Xiaofei

**Comment by casualjim** [comment]
I want to have a first release out by january. Now this is largely a volunteering effort so I don't want to commit to an actual date, for as long as there aren't many people contributing to this repository.
It says the project is sponsored by vmware, but that mostly means that I get to work on it from time to time during working hours. I have other responsibilities there too so it's not like they let me work in it full-time.

I've made 3 initial milestones for this project and I'm still working on the first one, which is arguably the hardest one because it needs to get the spec implementation ready.
At this point the API is largely stable, there are a few things to iron out in models code generation, and as you're aware certain things are passing validation and they don't necessarily have to.

So in short, I'm looking for help :) the more help I get the quicker the releases come. Submitting bugs is already great help. 

https://github.com/go-swagger/go-swagger/milestones

With the exception of showstoppers the issues for the "first release milestone" are frozen.

**Comment by casualjim** [comment]
There are binary releases available now. You can get swagger in a variety of ways without having to build it from source.
